rslib/src/scheduler/states/mod.rs

```diff
@@ -69,7 +68,8 @@ impl CardState {
 
 /// Info required during state transitions.
 pub(crate) struct StateContext<'a> {
-    pub fuzz_seed: Option<u64>,
+    /// In `[0, 1)`. Used to pick the final interval from the fuzz range.
```

Comment:
```suggestion
    /// Between 0.0-1.0. Used to pick the final interval from the fuzz range.
```

Reply:
I wanted to emphasise that the upper bound is not included, but I can use more intelligible phrasing.

Reply:
"In range 0.0..1.0"? Or we could use the more explicit Swift syntax:  Between 0.0..<1.0